config/serverless-operator.yaml

```diff
@@ -297,30 +297,46 @@ repositories:
     - olm-catalog/serverless-operator/index/Dockerfile
   e2e:
   - match: ^operator-e2e$
+    env:
+      SKIP_MESH_AUTH_POLICY_GENERATION: "true"
```

Comment:
Is it related to work on event plugin? Would it make more sense to check in S-O Makefile `if PROW` and based on that define defaults skips? Maybe we will need more skips in the future.

Reply:
This is a rather messy env var introduced in https://github.com/openshift-knative/hack/pull/77

Now, it is added to every command. I didn't like that, so I added support for defining the environmental variables.

I believe this variable makes sense only in terms of S-O, that's why I added it just to its config. Is it right, @skonto?